include/triton/Conversion/TritonGPUToLLVM/Utility.h

```diff
@@ -589,7 +578,7 @@ lowerLocalLdSt(Location loc, MLIRContext *ctx,
                ArrayRef<Value> valsArray, // Input for store, empty for load
                Type llvmElemTy, triton::gpu::MemDescType srcTy,
                SharedMemoryObject smemObj, ConversionPatternRewriter &rewriter,
-               const TargetInfoBase &targetInfo);
+               const TargetInfoBase &targetInfo, Operation *op = nullptr);
```

Comment:
Nit: rather than name it as a very generic `op`, what about using a more self-documenting name reflecting what's expected here?